__tests__/pages/home.test.js

```diff
@@ -35,7 +36,7 @@ describe('Home page', () => {
 
   it('should render the page', () => {
     render(<Home locale="en" content={content} />)
-    const heading = screen.getByRole('heading')
+    const heading = screen.getByText(en.pageHeading.title)
```

Comment:
This is going to have to change again when we move to the CMS; is there a more permanent fix maybe? (We don't care that the heading exists per say, but that the page rendered...)

Reply:
Yea I was thinking about that. I would of used data-test-id by Design System does not support it. I have an idea.